test/frontend/UserProfile.test.js

```diff
@@ -0,0 +1,82 @@
+/* global describe it */
+import 'jsdom-global/register'
+import React from 'react'
+import Enzyme, { mount } from 'enzyme'
+import { expect } from 'chai'
+import 'regenerator-runtime/runtime'
+
+import UserProfile from '../../src/Pages/Profile/MemberView/Profile'
+import InfoCard from '../../src/Pages/Profile/MemberView/InfoCard'
+import Adapter from 'enzyme-adapter-react-16'
+
+import winter from '../../src/Pages/Profile/MemberView/Image/winter'
+import fall from '../../src/Pages/Profile/MemberView/Image/fall'
+import summer from '../../src/Pages/Profile/MemberView/Image/summer'
+import spring from '../../src/Pages/Profile/MemberView/Image/spring'
+
+Enzyme.configure({ adapter: new Adapter() })
+
+describe('<UserProfile />', () => {
+  const user = {
+    token: 'token'
+  }
+
+  let wrapper = mount(<UserProfile user={user} />)
+
+  it('Should render a <InfoCard /> component with one child', () => {
+    expect(wrapper.find(InfoCard)).to.have.lengthOf(1)
+  })
+
+  it('Should render a <img /> component with 3 children', () => {
+    expect(wrapper.find('img')).to.have.lengthOf(3)
+  })
+
+  it('Should render a <img /> where src = public/images/SCE-glow.png', () => {
+    expect(wrapper.find('img').get(0).props.src).equals('images/SCE-glow.png')
+  })
+
+  it('Should render 2 <img /> where src = spring, winter, fall, or summer; bases from month', () => {
+    let date
+    const RealDate = Date
+
+    function mockDate (isoDate) {
```

Comment:
A better way to mock this would be to use sinon. We already have it as a dependency and an example of me using it is in #190 

e.g.
```js
describe('<UserProfile />', () => {
  var stub = null
  before(done => {
    // mock the getMonth date to return values we want
    stub = sinon.stub(Date, 'getMonth')
    stub.onCall(0).returns(1) // return the month of january
    stub.onCall(1).returns(3) // return the month of april
    stub.onCall(2).returns(6) // return august
    stub.onCall(3).returns(8) // return september
    done()
  })
  after(done => {
    // get rid of the stub
    if (stub) stub.restore()
    done()
  })
  // rest of your tests
})
```
Now, the first time you mount `UserProfile`, it should render winter from your mocked date. The next time should be spring, the next time... you get the idea.

Read more about `sub.onCall()` [here](https://sinonjs.org/releases/latest/stubs/#stuboncalln-added-in-v18)